trino/trino.go

```diff
@@ -678,21 +699,38 @@ func (st *driverStmt) QueryContext(ctx context.Context, args []driver.NamedValue
 		stmt:    st,
 		queryID: sr.ID,
 		nextURI: sr.NextURI,
+		statsCh: st.statsCh,
+		doneCh:  st.doneCh,
 	}
 	if err = rows.fetch(false); err != nil {
 		return nil, err
 	}
+	st.doneCh <- struct{}{}
```

Comment:
Close this channel.

Reply:
added `close(st.doneCh)` on line after this.